packages/web/src/scss/components/FileUploader/index.html

```diff
@@ -173,14 +270,14 @@ <h2 class="docs-Heading">Input with Attachment</h2>
             <span class="FileUploaderInput__dragAndDropLabel">or drag and drop here</span>
           </label>
           <div class="FileUploaderInput__helperText">Max file size is 10 MB</div>
-          <input type="file" id="fileUploadWithAttachments" name="attachment" class="FileUploaderInput__input" />
+          <input type="file" id="fileUploadWithAttachments" name="attachment6" class="FileUploaderInput__input" data-spirit-element="input" />
```

Comment:
This is just nitpicking, no worries if you ignore this. Maybe we should avoid numbered items? I think we discussed it together. You change the order or add a new example and you should renumber everything :)

Reply:
Although it may seem that the naming does not make sense, in fact this name is extracted and used for the name of individual attachments as `attachments[]`.

So this name then disappears from the original input element